yarn.lock

```diff
@@ -2,2997 +2,3285 @@
 # yarn lockfile v1
```

Comment:
Why has changed in this file so much

Reply:
It might be the version of Yarn I was using, as I see it from the differences it added double quotes ("") to all dependency definitions.